snippets/base/migrations/0010_auto_20160405_1444.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from django_mysql.models.functions import AsType, ColumnAdd
+
+
+def migrate_aboutaccounts_snippets(apps, schema_editor):
+    """Migrate existing snippets that used about:accounts link filtering."""
+    Snippet = apps.get_model('base', 'Snippet')
+    Snippet.objects.filter(data__icontains='href=\\"about:accounts').update(
+        client_options=ColumnAdd('client_options', {'has_fxaccount': AsType('yes', 'CHAR')})
+    )
+    Snippet.objects.exclude(data__icontains='href=\\"about:accounts').update(
+        client_options=ColumnAdd('client_options', {'has_fxaccount': AsType('any', 'CHAR')})
+    )
```

Comment:
hey @akatsoulas I changed the migrations here to use `update()` to
1. be faster
2. preserve Snippet ordering b/c update does not alter `modified`

Also set default value for all snippets b/c before I was leaving out snippets without has_fxaccount value